janus_server/src/bin/janus_cli.rs

```diff
@@ -0,0 +1,307 @@
+use anyhow::{Context, Result};
+use base64::STANDARD_NO_PAD;
+use deadpool_postgres::Pool;
+use janus_core::time::{Clock, RealClock};
+use janus_server::{
+    binary_utils::{janus_main, BinaryContext, BinaryOptions, CommonBinaryOptions},
+    config::{BinaryConfig, CommonConfig},
+    datastore::{self, Datastore},
+    task::Task,
+};
+use k8s_openapi::api::core::v1::Secret;
+use kube::api::{ObjectMeta, PostParams};
+use rand::{thread_rng, Rng};
+use ring::aead::AES_128_GCM;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::BTreeMap,
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+use structopt::StructOpt;
+use tokio::fs;
+use tracing::info;
+
+static SCHEMA: &str = include_str!("../../../db/schema.sql");
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    janus_main::<_, Options, Config, _, _>(RealClock::default(), |ctx| async move {
```

Comment:
We could tweak the trace subscriber config so that it does things better suited to a utility that will primarily run in operator shells instead of a Kubernetes cluster. For instance, most of its trace output should probably go to stderr, or maybe there's other nice formatters we could turn on. I'm happy to defer that to a future PR, so logging an issue is sufficient.

Reply:
Filed https://github.com/divviup/janus/issues/315.